backend/api/v1/endpoints/connections.py

```
@@ -35,14 +35,24 @@ def reject_connection(request_id: int, db: Session = Depends(get_db), current_us
     return {"message": "Connection rejected!"}
 
 @router.get("/connections")
-def list_connections(db: Session = Depends(get_db), current_user: dict = Depends(get_current_user)):
-    return get_connections(db, current_user["id"])
+def list_connections(db: Session = Depends(get_db), current_user: User = Depends(get_current_user)):
```

Ruff (0.8.2)

Missing return type annotation for public function `list_connections`

(ANN201)